### PR TITLE
Civ 3720 add call to get case from ccd

### PR DESCRIPTION
**Pull request by sampankumar**
Civ 3720 add call to get case from ccd
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-3720

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/routes/features/dashboard/dashboardController.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import * as express from 'express';
 import {CivilServiceClient} from '../../../app/client/civilServiceClient';
 import config from 'config';
 import {DASHBOARD_URL} from '../../urls';
-import {AppRequest, UserDetails} from 'models/AppRequest';
-import {getOcmcDraftClaims} from '../../../app/client/legacyDraftStoreClient';
+import {AppRequest} from 'models/AppRequest';
+//import {getOcmcDraftClaims} from '../../../app/client/legacyDraftStoreClient';
```
Comment (commitBlob) [inline src/main/routes/features/dashboard/dashboardController.ts:6]: can you please remove commented out lines from VSC

File: src/main/routes/features/dashboard/dashboardController.ts [hunk 2]
```diff
@@ -26,12 +26,12 @@ function renderPage(res: express.Response, claimsAsClaimant: DashboardClaimantIt
 const dashboardController = express.Router();
 
 dashboardController.get(DASHBOARD_URL, async function (req: AppRequest, res) {
-  const user: UserDetails = req.session.user;
+  //const user: UserDetails = req.session.user;
```
Comment (commitBlob) [inline src/main/routes/features/dashboard/dashboardController.ts:29]: remove this please

File: src/main/routes/features/dashboard/dashboardController.ts [hunk 3]
```diff
@@ -26,12 +26,12 @@ function renderPage(res: express.Response, claimsAsClaimant: DashboardClaimantIt
...
   /*This is a call to validate integration with legacy draft-store. This will have to be refined in the future
   to display the draft claims on the dashboard*/
   const claimsAsClaimant : DashboardClaimantItem[] = await civilServiceClient.getClaimsForClaimant(req);
   const claimsAsDefendant : DashboardDefendantItem[] = await civilServiceClient.getClaimsForDefendant(<AppRequest>req);
-  const claimDraftSaved = await getOcmcDraftClaims(user.accessToken);
+  const claimDraftSaved : any = undefined;
```
Comment (laperla11) [inline src/main/routes/features/dashboard/dashboardController.ts:34]: you need to revert this change to `const claimDraftSaved = await getOcmcDraftClaims(user.accessToken);`

File: src/main/modules/utilityService.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import {Claim} from '../../main/common/models/claim';
+import * as express from 'express';
+
+const civilServiceApiBaseUrl = config.get<string>('services.civilService.url');
+const civilServiceClient: CivilServiceClient = new CivilServiceClient(civilServiceApiBaseUrl);
+
+/**
+ * Gets civil claim response object with claim from draft store
+ * @param claimId
+ * @returns claim from redis or undefined when no there is no data for claim id
```
Comment (laperla11) [inline src/main/modules/utilityService.ts:14]: pls update this part

**Review by laperla11 — CHANGES_REQUESTED** [review]
failing unit test are required to be fixed;

`19:13:31  Test Suites: 159 failed, 73 passed, 232 total
19:13:31  Tests:       454 passed, 454 total
19:13:31  Snapshots:   0 total
19:13:31  Time:        554.51s
19:13:31  Ran all test suites.
19:13:31  error Command failed with exit code 1.
19:13:31  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

File: src/main/modules/utilityService.ts (hunk 4 above)
Comment (sampankumar) [reply src/main/modules/utilityService.ts]: done

File: src/main/routes/features/dashboard/dashboardController.ts (hunk 3 above)
Comment (sampankumar) [reply src/main/routes/features/dashboard/dashboardController.ts]: Done

File: src/main/routes/features/dashboard/dashboardController.ts (hunk 2 above)
Comment (sampankumar) [reply src/main/routes/features/dashboard/dashboardController.ts]: Reverted back. I had to do this to get my local wokring, I accidentally pushed the code I shouldn't have

File: src/main/routes/features/dashboard/dashboardController.ts (hunk 1 above)
Comment (sampankumar) [reply src/main/routes/features/dashboard/dashboardController.ts]: Reverted back. I had to do this to get my local wokring, I accidentally pushed the code I shouldn't have

**Review by commitBlob — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]
